doc/source/report/plots/plot_distances.py

```diff
@@ -7,27 +8,17 @@
 numbins = 30
 xlim = [0, 40]
 col = 'blue'
-verbose = False
-step = 50
 
 plt.style.use('ggplot')
 plt.style.use('seaborn-notebook')
 
 # Extract strain data
-dist_strain = []
-strain = 0
-dist = distances_bystrain(strain, step=step)
-while dist.size > 0:
-    dist_strain.append(dist[dist >= threshold])
-    if verbose:
-        print('strain %s loaded.' % strain)
-    strain += 1
-    dist = distances_bystrain(strain, step=step)
+dist_strain = list(np.load('report/plots/distances.npy'))
```

Comment:
@jarrodmillman, one question: 
- Is it pointing to the right file or should I change the path? 
The plots are still not displayed in the rst file.